src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -210,6 +210,11 @@ bool IInventory.Verify(StoreView snapshot)
         public virtual bool Verify(StoreView snapshot, BigInteger totalSenderFeeFromPool)
```

Comment:
How about modify its use on `private void OnFillMemoryPool(IEnumerable<Transaction> transactions)`?

Reply:
No need, I just checked `OnFillMemoryPool` is an isolated case only. Not worth.